openshift/olm/knative-serving.catalogsource.yaml

```diff
@@ -280,19 +280,21 @@ data:
                         value: image-registry.openshift-image-registry.svc:5000/knative-serving/knative-serving-activator
                       - name: IMAGE_autoscaler
                         value: image-registry.openshift-image-registry.svc:5000/knative-serving/knative-serving-autoscaler
+                      - name: IMAGE_autoscaler-hpa
+                        value: image-registry.openshift-image-registry.svc:5000/knative-serving/knative-serving-autoscaler-hpa
                       - name: IMAGE_controller
                         value: image-registry.openshift-image-registry.svc:5000/knative-serving/knative-serving-controller
                       - name: IMAGE_networking-certmanager
                         value: image-registry.openshift-image-registry.svc:5000/knative-serving/knative-serving-certmanager
                       - name: IMAGE_networking-istio
                         value: image-registry.openshift-image-registry.svc:5000/knative-serving/knative-serving-istio
-                      - name: IMAGE_networking-nscert
+                      - name: IMAGE_networking-ns-cert
```

Comment:
I think it should be `nscert`? 
Because i see directory structure has [name](https://github.com/openshift/knative-serving/tree/release-v0.9.0/openshift/ci-operator/knative-images) as nscert
Correct me if i am wrong

Reply:
The variable here correlates with the name of the deployment, which is `networking-ns-cert`.

https://github.com/knative/serving/blob/master/config/networking-ns-cert.yaml#L18